packages/react-pagination/README.md

```diff
@@ -0,0 +1,11 @@
+# `@wpmudev/react-pagination`
```

Comment:
@veljkocukic don't forget to write the documentation for the package, that way developers will know how to use it. You can find a good example here:
[https://www.npmjs.com/package/@wpmudev/react-button](https://www.npmjs.com/package/@wpmudev/react-button)